### PR TITLE
Add additional text into README.md

### PR DESCRIPTION
**Pull request by cballrun**

### CODE REVIEW

**Comment by cballrun** [comment]
This is a comment

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# This is some text!!!
\ No newline at end of file
+# This is some text!!!
```
Comment (cballrun) [inline README.md:1]: comment on the edited file

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+# Here is some more text!
```
Comment (coopercodex) [inline README.md:3]: This is another comment!

Comment (coopercodex) [inline README.md:3]: This is yet another comment.

**Review by coopercodex — COMMENTED** [review]
this is a add on

**Pull request merged**